zrml/neo-swaps/src/tests/buy.rs

```diff
@@ -312,44 +312,50 @@ fn buy_fails_on_amount_out_below_min() {
             vec![_1_2, _1_2],
             CENT,
         );
-        let amount_in = _1;
-        assert_ok!(AssetManager::deposit(BASE_ASSET, &BOB, amount_in));
-        // Buying 1 at price of .5 will return less than 2 outcomes due to slippage.
+        let amount_in = _10;
+        #[cfg(not(feature = "parachain"))]
+        let expected_error = pallet_balances::Error::<Runtime>::InsufficientBalance;
+        #[cfg(feature = "parachain")]
+        let expected_error = orml_tokens::Error::<Runtime>::BalanceTooLow;
+        assert_ok!(AssetManager::deposit(BASE_ASSET, &BOB, amount_in - 1));
         assert_noop!(
             NeoSwaps::buy(
                 RuntimeOrigin::signed(BOB),
                 market_id,
                 2,
                 Asset::ScalarOutcome(market_id, ScalarPosition::Long),
                 amount_in,
-                _2,
+                0,
             ),
-            Error::<Runtime>::AmountOutBelowMin,
+            expected_error,
         );
     });
 }
 
 #[test]
-fn buy_fails_on_spot_price_above_max() {
+fn buy_fails_on_amount_out_below_min() {
     ExtBuilder::default().build().execute_with(|| {
         let market_id = create_market_and_deploy_pool(
             ALICE,
             BASE_ASSET,
-            MarketType::Categorical(2),
+            MarketType::Scalar(0..=1),
```

Comment:
Why scalar here? Does this have a particular meaning? Should you rename the test name for it?

Reply:
Doesn't matter which type of market we use. No idea how this change came about.